api/src/main/java/uk/gov/dhsc/htbhf/claimant/communications/UpcomingBirthdayEmailHandler.java

```diff
@@ -50,7 +54,7 @@ public void sendChildTurnsFourEmail(PaymentCycle paymentCycle, NextPaymentCycleS
         EmailMessagePayload messagePayload = buildChildTurnsAgeNotificationEmailPayload(paymentCycle,
                 entitlement, multipleChildrenTurningFourInNextMonth, EmailType.CHILD_TURNS_FOUR);
         log.info("Sending email for child turns 4 for Payment Cycle after cycle with id: [{}]", paymentCycle.getId());
```

Comment:
Is it worth updating this message to say that its going to be delayed? It could be either here or in `MessageQueueClient`.

Reply:
Added comment.